Symfony/CS/Tests/Fixer/PSR2/BracesFixerTest.php

```diff
@@ -916,14 +948,16 @@ public function CCCC()
             ),
             array(
                 '<?php
-    filter(function   ($a) {});',
+    filter(function   ($a) {
+    });',
                 '<?php
     filter(function   ($a)
     {});',
             ),
             array(
                 '<?php
-    filter(function   ($b) {});',
+    filter(function   ($b) {
+    });',
```

Comment:
why changing this ? It should be possible to keep no-op closures on a single line IMO.